charts/gateway-helm/templates/envoy-gateway-deployment.yaml

```diff
@@ -84,7 +84,7 @@ spec:
         - mountPath: /certs
           name: certs
           readOnly: true
-      {{- include "eg.image.pullSecrets" . | nindent 6 }}
+      imagePullSecrets: {{- include "eg.image.pullSecrets" . | nindent 8 }}
```

Comment:
can we revert this ?
it seems to be changing the default indentation style in the file
https://github.com/envoyproxy/gateway/pull/5719/files#r2047900977